.github/workflows/on-pullrequest-dockerfile.yml

```diff
@@ -0,0 +1,78 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: On Pull Request - Dockerfile
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths:
+      - ".github/workflows/on-pullrequest-dockerfile.yml"
+      - "src/*"
+  workflow_dispatch:
+
+env:
+  DOCKER_FILE_DIR: src
+  DOCKER_USERNAME: ${{ vars.DOCKER_USERNAME }}
+  DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+jobs:
+  docker-analysis:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+      pull-requests: write
+    strategy:
+      fail-fast: false
+      matrix:
+        dockerfile:
+          [
+            { name: "publisher", path: "src/Dockerfile", type: "local" },
```

Comment:
The `type` variable can be removed here because it's not needed

Reply:
type removed